src/Features/VisualBasic/Portable/Completion/VisualBasicCompletionService.vb

```diff
@@ -20,13 +21,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion
         Friend Class Factory
             Implements ILanguageServiceFactory
 
+            Private ReadOnly _listenerProvider As IAsynchronousOperationListenerProvider
+
             <ImportingConstructor>
             <Obsolete(MefConstruction.ImportingConstructorMessage, True)>
-            Public Sub New()
+            Public Sub New(Optional listenerProvider As IAsynchronousOperationListenerProvider = Nothing)
```

Comment:
The C# side doesn't have a similar optional pattern, can we make this match? I'm not sure if this would get flagged if VB had nullable analysis.

Reply:
Fixed